src/mito2/src/sst/parquet.rs

```diff
@@ -15,7 +15,7 @@
 //! SST in parquet format.
 
 mod format;
-mod helper;
+pub mod helper;
```

Comment:
`pub(crate)` should be enough. Do not expose too many details unless it is necessary.